scripts/hypothesistooling.py

```diff
@@ -149,12 +150,19 @@ def create_tag_and_push():
         'git push ssh-origin --tags',
         'git push ssh-origin HEAD:master',
     ]:
-        subprocess.check_call([
-            'ssh-agent', 'sh', '-c',
-            'chmod 0600 deploy_key && ' +
-            'ssh-add deploy_key && ' +
-            command
-        ])
+        for retries_left in range(2, -1, -1):
+            try:
+                subprocess.check_call([
+                    'ssh-agent', 'sh', '-c',
+                    'chmod 0600 deploy_key && ' +
+                    'ssh-add deploy_key && ' +
+                    command
+                ])
+                break
```

Comment:
I confess I never remember try/else clauses are a thing in Python (which is weird because I use for/else clauses a lot and I'm one of the few people who do).

I'm going to be lazy and leave this as is, but I'll try to remember for next time.